workspaces/ui/src/components/diff/v2/DiffReviewPage.js

```diff
@@ -290,7 +287,7 @@ const useStyles = makeStyles((theme) => ({
     paddingLeft: 20,
     paddingRight: 20,
     paddingTop: 20,
-    paddingBottom: 400,
+    paddingBottom: 100,
```

Comment:
Great, that gets rid of a whole bunch of arbitrary white space. Have you seen the `theme.spacing(x)` API? I'm trying to get us all to use whole numbers of that for any spacing between items, so our white spaces become very nicely consistent throughout the entire app.

Reply:
worked perfectly — switched to `theme.spacing(1)`, since it really just needed a minimal amount of padding because otherwise the card was getting a little cut off 
<img width="759" alt="Screen Shot 2020-08-05 at 9 46 46 AM" src="https://user-images.githubusercontent.com/9388431/89440522-95b72100-d700-11ea-8492-682a60f73724.png">

Reply:
Excellent!